test/parallel/test-assert-partial-deep-equal.js

```diff
@@ -713,7 +723,7 @@ describe('Object Comparison Tests', () => {
       {
         description: 'compares two Uint8Array objects',
         actual: { typedArray: new Uint8Array([1, 2, 3, 4, 5]) },
-        expected: { typedArray: new Uint8Array([1, 2, 3]) },
+        expected: { typedArray: new Uint8Array([1, 2, 3, 5]) },
```

Comment:
Does this PR change the behavior? Is this change necessary for the test to pass?

Reply:
It does not change and passes both cases. I wanted to trigger a specific code line and make sure the test still passes.

Reply:
Thanks for clarifying @BridgeAR! :)